DEVELOP_V2.md

```diff
@@ -139,6 +139,10 @@ In order to update the Go version, you need to change the version in the followi
 In order to update the thrift version, you need to change the version in the following places:
 
 1. ./commons/Dockerfile
+   1. After updating Dockerfile you need to re-build the docker image and push it to the registry with:
+   ```shell
+   make docker/publish
```

Comment:
Maybe we should mention here to run the github workflow. I would avoid pushing images from local machine